flexxamples/howtos/openlayers.py

```diff
@@ -0,0 +1,185 @@
+# doc-export: OpenlayersExample
+"""
+Example demonstrating how to create a simple map app with Openlayers and flexx.
+
+Features:
+
+* Load geojson data to map.
+* Draw points on map.
+* Working with Openlayers events.
+
+"""
+
+from flexx import flx
+import os
+
+BASE_DIR = os.getcwd()
+
+with open(BASE_DIR + '/static/js/data.json') as f:
+    geojson = f.read()
+
+flx.assets.associate_asset(__name__, 'https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.css')
```

Comment:
The line is too long. I thing this should do:
```suggestion
flx.assets.associate_asset(__name__,
    'https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.css')
```